paymentsheet/src/test/java/com/stripe/android/customersheet/state/DefaultCustomerSheetLoaderTest.kt

```diff
@@ -617,56 +388,6 @@ class DefaultCustomerSheetLoaderTest {
         assertThat(result.exceptionOrNull()).isInstanceOf<IllegalStateException>()
     }
 
-    @Test
-    fun `On failed to load elements session, show report error`() = runTest {
```

Comment:
Do we have tests for these elsewhere?

Reply:
They are covered in the `CustomerAdapterDataSource` & `CustomerSessionInitializationDataSource` since they report elements session failures.